### ISSUE TITLE
Improve early anti-air BO

### ISSUE THREAD

**Issue by RaphaelRoyerRivard** [RaphaelRoyerRivard/MicroMachine#320]
Otherwise we will use our resources to build an engineering bay and turrets before starting our first banshee...

**Comment by Unusual229** [comment]
I don't really agree, I'd rather not have a condition that forces this.
We could delay the turret range upgrade or not do it (since I am not sure how useful it is anyway..)

**Comment by RaphaelRoyerRivard** [comment]
Maybe instead of only delaying our Engineering Bay for after we queued our first Banshee, we could start with a Viking, then a Banshee and then the Engineering Bay.

**Comment by Unusual229** [comment]
Logic could be :
If we have an idle starport, queue viking as high priority
  then queue engineering bay + turrets as low priority
else queue engineering bay + turrets as high priority

The banshee will be queued whenever the starport is idle.

**Comment by Unusual229** [comment]
Is the BO good now?

**Comment by RaphaelRoyerRivard** [comment]
Yes

Raphaël Royer-Rivard
